Add i18n locale information for Uganda, Kenya and Tanzania.

bahiirwa opened the pull request:
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:
Add i18n locale information for Uganda, Kenya and Tanzania.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

<!-- Mark completed items with an [x] -->

### Changelog entry
Add i18n locale information for Uganda, Kenya and Tanzania.

Commit:
Add Uganda Details

Commit:
Add Kenya Details

Commit:
Add Tanzania Details

rrennick reviewed (dismissed):
This looks good other than needing to be rebased.

bahiirwa committed:
Merge branch 'master' into locales

bahiirwa commented:
Rebase done @rrennick

rrennick commented on i18n/locale-info.php:
```diff
@@ -162,13 +162,21 @@
+		'decimal_sep'    => '.',
+		'num_decimals'   => 0,
+    'weight_unit'    => 'kg',
+		'dimension_unit' => 'cm',
```
This looks like it's missing the `),` to close the array.

rrennick commented on i18n/locale-info.php:
```diff
@@ -162,13 +162,21 @@
 		'decimal_sep'    => '.',
 		'num_decimals'   => 2,
-		'weight_unit'    => 'kg',
+    'weight_unit'    => 'kg',
```
This looks like a tab has been replaced with spaces.

bahiirwa committed:
Fix Spacing and missing bracket

- [x] Fix Spacing to project tab delimiter
- [x] Add missing bracket after the conflict merge.

bahiirwa committed:
PHPCS Fix

bahiirwa commented:
The E2E tests passed the first time and failed when I made a PHPCS change. This PR does not have any code to E2E effect.

rodrigoprimo reviewed (dismissed):
Thanks for your contribution.

rodrigoprimo committed:
Merge branch 'master' into locales
